### PR TITLE
Added table of contents to RULES.md for easier navigation

### PR DESCRIPTION
**Pull request by ajorpheus**
Closes #161

### CODE REVIEW

File: docs/RULES.md [hunk 1]
```diff
@@ -1,3 +1,48 @@
+
+<!-- vim-markdown-toc GFM -->
```
Comment (psyomn) [inline docs/RULES.md]: I'd like to keep things editor agnostic. Would it be cool to remove this?

**Review by psyomn — COMMENTED** [review]
Thanks for the PR! There was another PR that did something similar: #161 (so I'll have to remember to close that one once this is merged).

Is there a tool you have that can be integrated in the build/release process? I'd rather have something that generates the table rather than having to hand write it each time there are modifications to the rules.

File: docs/RULES.md (hunk 1 above)
Comment (ajorpheus) [reply docs/RULES.md]: Yes sure, go ahead. That is only meant to demarcate the toc area for a VIM plugin that does the heavy lifting of generating the toc. Please go ahead and remove it.

**Comment by ajorpheus** [comment]
Have removed the placeholder.

I agree that it would be nice to have generating or updating of toc automated. Off the top of my head, I don't know of anything which might automate this.

**Pull request merged**